2.5 compatibility - code cleanup - french translation

pioc92 opened the pull request:
Hi, i updated the code for 2.5 compatibility. I replaced old functions by new ones. I updated french translation.

pioc92 committed:
yaga starting fork

mise à jour application for 2.5 update

pioc92 committed:
translate and stop spidertrap

Translate some french
Remove link to best content in activity tab because of a problem of infinite links i have no time to correct

pioc92 committed:
maj deprecated function ConsolidateArrayValuesByKey

update deprecated function ConsolidateArrayValuesByKey
replace by array_column

hgtonight commented on settings/class.hooks.php:
```diff
@@ -256,9 +256,9 @@ public function ProfileController_Best_Create($Sender, $UserReference = '', $Use
    */
   public function ProfileController_AddProfileTabs_Handler($Sender) {
-    if(is_object($Sender->User) && $Sender->User->UserID > 0) {
+    /*if(is_object($Sender->User) && $Sender->User->UserID > 0) {
```
Is there a reason you left this in commented?

pioc92 replied:
It’s for SEO problem i think. Googlebot crawle dans these urls and indexed them. You can remove the comments.

pioc92 replied:
Spider trap too when i crawled my website with screaming frog

hgtonight replied:
If the code should not be there, the entire hook should be removed. In this patch how does a user get to their profile best content?

pioc92 replied:
Yes remove comments, i will search another solution

hgtonight replied:
Feel free to play with it and push more commits to this branch

bleistivt replied:
@pioc92 That page already has `robots: noindex/noarchive`. This is not a SEO problem. You can just hide the link with CSS if you don't like it.

pioc92 replied:
@bleistivt when you crawl the forum, the crawler can't stop because of this link. Try it with screaming frog!

pioc92 committed:
fixed unable to add a new badge

Fixed unable to add a badge because of a javascript error in gamification side menu

pioc92 committed:
fix badges count

Fix badges count before badge label in user info

pioc92 committed:
Add noindex/noarchive to /best controller

stop duplicate content

pioc92 committed:
fix notices, warnings and other problems

thank breivist

pioc92 committed:
fix settings error

Fix setting blank page after enabling application

pioc92 committed:
Fix DATE_ISO8601

Replace DATE_ISO8601 by Gdn_Format::toDateTime() / Thank bleistivt

pioc92 committed:
Fix DATE_ISO8601 mysql error

thanks bleistivt

pioc92 committed:
fix sql strict mode error

add a group by to enable mysql strict mode

pioc92 committed:
fix infinite best profile content

On profile page, best content hadn't limit to results so there was an infinite url paging system

pioc92 committed:
fix mysql strict errors

pioc92 committed:
fix mysql strict errors

pioc92 committed:
fix best profiles unlimited pagination

pioc92 committed:
Merge branch 'master' of https://github.com/pioc92/Application-Yaga

pioc92 committed:
noindex best pages

pioc92 committed:
update yaga views for new keystone theme in vanilla 2.8

pioc92 committed:
update fr locale

pioc92 committed:
fix count item width on mobile phone

pioc92 committed:
fix parse error

pioc92 committed:
fix parse error

pioc92 committed:
change settings button's class

pioc92 committed:
change button's class

pioc92 committed:
Update class.hooks.php

fix configuration backup path

pioc92 committed:
fix hardcoded link to routeur4g.fr user profiles

pioc92 committed:
fix settings save error

pioc92 committed:
fix GetUserCount double declaration
